fmriprep/cli/run.py

```diff
@@ -168,9 +168,9 @@ def get_parser():
 
     # FreeSurfer options
     g_fs = parser.add_argument_group('Specific options for FreeSurfer preprocessing')
-    g_fs.add_argument('--fs-no-reconall', '--no-freesurfer',
+    g_fs.add_argument('--fs-no-reconall',
                       action='store_false', dest='run_reconall',
-                      help='disable FreeSurfer preprocessing')
+                      help='disable FreeSurfer recon-all preprocessing')
```

Comment:
Can you make this "surface" instead of "recon-all"? As noted above, it also disables bbregister and mri_vol2surf.